static/src/javascripts/projects/common/modules/experiments/tests/tailor-survey.js

```diff
@@ -107,6 +101,12 @@ const inArticleWriter = (survey, surveyId) =>
         return surveyId;
     });
 
+// Getting simple json from tailor's response to be passed to the html template
+const getJsonFromSurvey = survey => ({
```

Comment:
Could be written as...

`const getJsonFromSurvey = ({question, id}) => ({question, id});`